test/src/main/java/io/strimzi/test/annotations/ParallelParameterizedTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.test.annotations;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.api.parallel.ResourceAccessMode;
+import org.junit.jupiter.api.parallel.ResourceLock;
+import org.junit.jupiter.params.ParameterizedTest;
+
+
+/**
+ * Annotation for running parallel parametrized tests in strimzi test suite
+ * please be sure that you know laws of parallel execution and concurrent programming
+ * be sure that you do not use shared resources, and if you use shared resources please work with synchronization
+ */
+@Target(ElementType.METHOD)
+@Retention(RUNTIME)
+@Execution(ExecutionMode.CONCURRENT)
+@ResourceLock(mode = ResourceAccessMode.READ, value = "global")
+@Documented
+@ParameterizedTest
+public @interface ParallelParameterizedTest {
```

Comment:
Why is this needed? I think we have already other parametrized tests where this was not needed.

Reply:
I can remove it, I just followed the pattern of parallel tests.

Reply:
Removed